### PR TITLE
added neon db to database.json

### LINKED ISSUE #2023: [Add] Neon DB to Database
### Site URL

https://neon.tech/

### Description of the above provided site

Neon is Serverless Postgres built for the cloud. Neon separates compute and storage to offer modern developer features such as autoscaling, branching, bottomless storage, and more.

### Category

Backend

### Sub Category

"Database"

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Hrithik0112**
## Fixes Issue

- Added Neon DB to into Database sub-category of Backend Category

Fixes #2023 

## Changes proposed
- updated database.json

## Screenshots

Screenshot 2023-10-19 003314

### CODE REVIEW

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

File: database/backend/database.json [hunk 1]
```diff
@@ -82,5 +82,12 @@
     "url": "https://mariadb.org/",
     "category": "backend",
     "subcategory": "database"
+  },
+  {
+    "name": "Neon DB",
+    "description": "Neon is Serverless Postgres built for the cloud. Neon separates compute and storage to offer modern developer features such as autoscaling, branching, bottomless storage, and more.",
```
Comment (CBID2) [inline database/backend/database.json:88]:
```suggestion
    "description": "Neon is Serverless Postgres built for the cloud. It separates compute and storage to offer modern developer features such as autoscaling, branching, bottomless storage, and more.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Hrithik0112! 😊 There are merge conflicts that need to be solved and your description needs a bit more tweaking. Other than that, you're off to a great start! 😊

**Comment by Hrithik0112** [comment]
@CBID2  I understand the problem, I guess you solved the issue, aur am I wrong? 
if I am so what I need to do know?
thank you for assistance.

**Comment by CBID2** [comment]
> @CBID2  I understand the problem, I guess you solved the issue, aur am I wrong? 
> 
> if I am so what I need to do know?
> 
> thank you for assistance.

Hi @Hrithik0112. I just gave feedback on the description and pointed out that you had merge conflicts. These occur when two or more people are working on the same file and make changes to the exact same lines of code. It's basically a technical version of a car crash.

**Comment by Hrithik0112** [comment]
@CBID2 have a look, is there something more to do, or it's done

**Comment by Hrithik0112** [comment]
@CBID2 can you help me advising, should create a new branch and make a new PR?

**Comment by Anmol-Baranwal** [comment]
@Hrithik0112 
You can literally search anywhere, how to solve the merge conflict. I will do it for you, but in good repositories. You may have to do it several times.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — APPROVED** [review]
Well done @Hrithik0112

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**